src/third-party/imgui/imstb_textedit.h

```diff
@@ -277,6 +277,10 @@
 #ifndef INCLUDE_STB_TEXTEDIT_H
 #define INCLUDE_STB_TEXTEDIT_H
 
+// NOLINTBEGIN
+#pragma GCC diagnostic push
```

Comment:
Moving the lint changes to a new PR would reduce noise (unless, of course, they are necessary to pass clang-tidy).

Reply:
I only included these changes because suddenly it started complaining about these files.